app/scripts/views/mixins/signin-mixin.js

```diff
@@ -105,16 +105,10 @@ define(function (require, exports, module) {
 
         if (verificationReason === VerificationReasons.SIGN_IN &&
             verificationMethod === VerificationMethods.EMAIL) {
-          return this.navigate('confirm_signin', {
-            account: account,
-            flow: this.flow
-          });
-        } else {
-          return this.navigate('confirm', {
-            account: account,
-            flow: this.flow
-          });
+          return this.navigate('confirm_signin', { account });
```

Comment:
Just for my own understanding, the flow model is now [taken care of by the `metrics` model](https://github.com/mozilla/fxa-content-server/pull/4718/files#diff-7b6eb0846bca35324d8b1acd38a32302R174), so no need to pass it from flow to flow?

What did the `flow` model provide to the confirmation screen? I was confused why [confirm had a reference to it](https://github.com/mozilla/fxa-content-server/pull/4718/files#r101043055) before, was it for the ResumeTokenMixin?